### ISSUE TITLE
Evaluation on custom test dataset

### ISSUE THREAD

**Issue by SPP3000** [RUCAIBox/RecBole#1609]
Dear RecBoleTeam,

I need to evaluate some models on a test dataset for which the hidden (to be recommended) items are predefined.
I am using currently Gru4Rec as my first model. Therefore, at least some interactions in the training set are listed for every user_id. I enjoy working with AtomicFiles. So is there an easy way to list the unseen interactions in a test file
or can I predict the next k items for a subset of users and evaluate the prediction on my own ground truth with the RecBole metrics in some way?

Thank you in advance for your help.

**Comment by SPP3000** [comment]
Are there any updates regarding my question?

**Comment by Sherry-XLL** [comment]
Hello, @SPP3000! Thanks for your attention to RecBole and sorry for my late reply. 

If you want to evaluate models based on your own pre-split datasets, you can use [benchmark_filename](https://recbole.io/docs/user_guide/config/data_settings.html) to load your train, valid, test dataset. In this way, you can customize your own datasets and evaluate on your test set.

For more details about the sample code for running session-based recommendation benchmarks using RecBole, you can refer to [RecBole/run_example/session_based_rec_example.py](https://github.com/RUCAIBox/RecBole/blob/master/run_example/session_based_rec_example.py) with sample sequential datasets (diginetica-session/tmall-session/nowplaying-session).
